docs/reference/api/platform/Callback.md

```diff
@@ -2,7 +2,7 @@
 
 A callback is a user provided function that a user may pass to an API. The callback allows the API to execute the user’s code in its own context.
 
-This is the technical reference for callbacks. You should read the [introduction to callbacks](/docs/v5.6/reference/platform-overview.html#callbacks) first to understand how to use them.
+This is the technical reference for callbacks. You should read the [introduction to callbacks on the overview page](/docs/v5.6/reference/platform-overview.html#callbacks) first to understand how to use them.
```

Comment:
I was thinking more along the lines of:

This is the technical reference for callbacks. Read the [Callbacks](/docs/v5.6/reference/platform-overview.html#callbacks) section of the [Platform Overview](/docs/v5.6/reference/platform-overview.html) for deeper insight on their use.

Reply:
@loverdeg-ep Thanks for the feedback. In the future, if you have specific phrasing that you think would be best for our documentation, please don't hesitate to put up a Pull Request (PR) with the change you'd like to see made. We love receiving PRs from our community members and try to merge them quickly.

Reply:
@AnotherButler 
No problem. 
I wasn't able to dig into https://github.com/ARMmbed/documentation_docs before it went private. 
Didn't want to make any direct contributions without knowing your documentation standards.